### PR TITLE
A few minor fixes

### PR DESCRIPTION
**Pull request by avagin**

### CODE REVIEW

**Comment by LK4D4** [comment]
LGTM
But jenkins is not working :)

**Comment by crosbymichael** [comment]
Looks like you need to gofmt some files

**Comment by avagin** [comment]
@crosbymichael @LK4D4  thanks. I have uploaded a fixed version.

File: process.go [hunk 1]
```diff
@@ -54,8 +54,12 @@ func (p Process) Wait() (*os.ProcessState, error) {
 
 // Pid returns the process ID
 func (p Process) Pid() (int, error) {
+	// A random negative values can be dengerous, because
+	// they can be valid for the kill system call.
+	const errorPid = -(int(^uint(0)>>1) - 1) // MinInt
```
Comment (mrunalp) [inline process.go]: Can we use a constant from math package here?

**Comment by avagin** [comment]
@mrunalp Done. Thanks.

**Comment by mrunalp** [comment]
The kernel seems to have a max limit of 2^22 - 1 = 4194303 for pid_max. There couldn't be more process groups than that so this should actually be safe as math.MinInt32 is -2147483648

**Comment by dqminh** [comment]
> The kernel seems to have a max limit of 2^22 - 1 = 4194303 for pid_max. There couldn't be more process groups than that so this should actually be safe as math.MinInt32 is -2147483648

Yes i think you are correct. TIL that :heart: 
(https://github.com/torvalds/linux/blob/master/include/linux/threads.h#L33 )

LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**